test/parallel/test-handle-wrap-close-abort.js

```diff
@@ -1,16 +1,19 @@
 'use strict';
-require('../common');
+const common = require('../common');
 
-process.on('uncaughtException', function() { });
+function noop() {}
```

Comment:
Just curious why you decided to define `noop` and `uncaughtExceptionHandler`, instead of just doing `common.mustCall(function() {}, 2)` inline.